modules/check_in/app/services/check_in/vaos/appointment_service.rb

```diff
@@ -15,7 +15,7 @@ def get_appointments(start_date, end_date, statuses = nil)
 
         with_monitoring do
           response = perform(:get, appointments_base_path, params, headers)
-          Oj.load(response.body).with_indifferent_access
+          response.body
```

Comment:
Wondering, if we can return both `http status` & `http body` from the `get_appointments` method, that way it will be convenient to handle other statuses returned from the VAOS endpoint?

Reply:
any request status other than 2xx would throw an exception, and will return an error response. This way, it's easier to pass the response to serializer in the controller and return it to the caller.